DashAISoftware/DashAI
# Back/modify db

# Summary


Add input and output column names to Experiment model on DB.

## Type of change



- Back end new feature.

## Changes


- Merge with staging branch.
- Add input and output columns to use on New Experiment flow.

## How to Test


1.- Go to the root of the project
2.- Run ```pip install -r requirements.txt```
3.- Run ```alembic upgrade head to apply migrations scripts```
4.- Run ```alembic current``` to check the current migration status
